tests/test_basics.py

```diff
@@ -280,6 +296,9 @@ def check_tai_from_utc(self, utc_ap):
         """
         tai = salobj.tai_from_utc(utc_ap.utc.unix)
 
+        tai_alt = alternate_tai_from_utc_unix(utc_ap.utc.unix)
+        self.assertAlmostEqual(tai, tai_alt, delta=1e-6)
+
```

Comment:
Can you add a couple of failing tests? One for 1970 and one for the year 3000? I can't see anywhere where you test that out of range dates will fail.

Reply:
Yes. Excellent suggestion.